plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueInputConverter.java

```diff
@@ -51,15 +54,21 @@ public static DatabaseInput convertDatabase(Database database)
 
     public static TableInput convertTable(Table table)
     {
+        Optional<String> comment = Optional.ofNullable(table.getParameters().get(TABLE_COMMENT));
+        Map<String, String> tableParameters = table.getParameters().entrySet().stream()
+                .filter(entry -> !entry.getKey().equals(TABLE_COMMENT))
+                .collect(toImmutableMap(Map.Entry::getKey, Map.Entry::getValue));
+
         TableInput input = new TableInput();
         input.setName(table.getTableName());
         input.setOwner(table.getOwner().orElse(null));
         input.setTableType(table.getTableType());
         input.setStorageDescriptor(convertStorage(table.getStorage(), table.getDataColumns()));
         input.setPartitionKeys(table.getPartitionColumns().stream().map(GlueInputConverter::convertColumn).collect(toImmutableList()));
-        input.setParameters(table.getParameters());
+        input.setParameters(tableParameters);
         table.getViewOriginalText().ifPresent(input::setViewOriginalText);
         table.getViewExpandedText().ifPresent(input::setViewExpandedText);
+        comment.ifPresent(input::setDescription);
```

Comment:
this has 2k max length, while params have 512k limit

let's separate addition of `commentTable` support from other changes

Reply:
Split out the column comment changes to https://github.com/trinodb/trino/pull/19076